src/datreant/core/util.py

```diff
@@ -1,19 +1,19 @@
 import os
 
 
-def makedirs(path):
+def makedirs(path, mode=511, exists_ok=False):
```

Comment:
Plus this method is meant to mimic `os.makedirs` (sp?) which has `exists_ok=True` as default